dlls/monsters.cpp

```diff
@@ -2540,18 +2540,16 @@ float CBaseMonster::ChangeYaw ( int yawSpeed )
 	if (current != ideal)
 	{
 		if( m_flLastYawTime == 0 )
-		{
 			m_flLastYawTime = gpGlobals->time - gpGlobals->frametime;
-		}
 
 		float delta = gpGlobals->time - m_flLastYawTime;
 
 		m_flLastYawTime = gpGlobals->time;
 
-		if( delta > 0.25 )
+		if (0.25 <= delta || delta <= 0)
```

Comment:
I believe this patch can be condensed to just adding `|| delta < 0` to this check?

Reply:
yeah, i was syncing it up to valve's version with "<=" in the A condition, but sure